gulpfile.js

```diff
@@ -0,0 +1,40 @@
+const spawn = require('cross-spawn');
+const gulp = require('gulp');
+const sourcemaps = require('gulp-sourcemaps');
+const ts = require('gulp-typescript');
+const path = require('path');
+
+const tsProject = ts.createProject('tsconfig.json');
+
+gulp.task('compile', () => {
+  return tsProject.src()
+    .pipe(sourcemaps.init())
+    .pipe(tsProject())
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('dist'));
+});
+
+gulp.task('copy_html', () => {
+  return gulp.src(['lib/*.html'])
+    .pipe(gulp.dest('dist'));
+});
+
+gulp.task('start', ['compile', 'copy_html'], () => {
+  return new Promise((resolve, reject) => {
+    spawn(getLocalPathForBinary('electron'), ['dist/app.js'], { stdio: 'inherit' })
```

Comment:
I'm going to open an issue to make this so we have hot recompile + reload. I was hacking that by having tsc . -watch in a separate terminal instance for now.